src/networks.json

```diff
@@ -1196,6 +1196,23 @@
     "start": 94085,
     "logo": "ipfs://QmaQxfwpXYTomUd24PMx5tKjosupXcm99z1jL1XLq9LWBS"
   },
+  "2222": {
+    "key": "2222",
+    "name": "Kava EVM",
+    "shortName": "KAVA",
+    "chainId": 2222,
+    "network": "mainnet",
+    "multicall": "0x30A62aA52Fa099C4B227869EB6aeaDEda054d121",
+    "rpc": [
+        "https://evm.kava.io",
+        "https://evm2.kava.io"
```

Comment:
Hi @gathonwar You don't have any other nodes? these nodes are too slow, they return a response after a 5secs-10secs delay,